src/administration/administration.controller.ts

```diff
@@ -0,0 +1,126 @@
+import {
+  Body,
+  Controller,
+  Get,
+  HttpException,
+  HttpStatus,
+  Post,
+  Put,
+  Req,
+  Res,
+  UseGuards,
+} from '@nestjs/common';
+import { AdministrationLoginDto } from './dtos/administration-login.dto';
+import { AdministrationService } from './administration.service';
+import { AdministrationDto } from './dtos/administration.dto';
+import { StudentDto } from '../student/dtos/student.dto';
+import {
+  AdminAuthorizationGuard,
+  StaffAuthorizationGuard,
+} from './guards/admin.guard';
+import { throttle } from 'rxjs';
+import { Aggregate } from 'mongoose';
+import { DepartmentDto } from '../department/dtos/departmentData.dto';
+
+@Controller('administration')
+export class AdministrationController {
+  constructor(private adminService: AdministrationService) {}
+
+  @Post('/login')
+  async login(@Body() body: AdministrationLoginDto, @Res() res) {
+    try {
+      const { administratorId, password } = body;
+      const loginResponse = await this.adminService.login(
+        administratorId,
+        password,
+      );
+      return res.status(HttpStatus.OK).send(loginResponse);
```

Comment:
Here if you don't want to specify status you can do below like this

 @Get('/login')
    @HttpCode(HttpStatus.OK)
    async login(

        @Body() body: AdministrationLoginDto ,
        @Res() res,
    )